src/main/resources/templates/testRules.html

```diff
@@ -1,212 +1,200 @@
-<!DOCTYPE html>
-<html xmlns:th="http://www.thymeleaf.org">
-
-<head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
-    <meta name="description" content="" />
-    <meta name="author" content="" />
-    <script type="text/javascript" src="js/testrules.js"></script>
-    <script type="text/javascript" src="js/downloadFile.js"></script>
-    <script type="text/javascript" src="resources/testRules_constants.js"></script>
-</head>
-
-<body>
-    <div class="container mw-100 pb-3" id="testRule">
-        <div class="text-center">
-            <h1 id="test_rules_header">Test Rules</h1>
-        </div>
-        <div class="row px-3">
-            <div class="p-1 col-lg-6 col-md-6 col-12 border border-right" id="testRulesDOMObject">
-                <div class="d-flex flex-wrap sticky-button-bar">
-	                <div class="col-11">
-	                   <button class="btn btn-primary add_rule mr-1 mt-1 d-inline-block" data-bind="click : $root.addRule.bind($data, ruleTemplate)">
-                           Add
-                           <i class="fa fa-fw  fa-plus"></i>
-                       </button>
-	                   <button class="btn btn-primary upload_rules mr-1 mt-1 d-inline-block">
-                           Load From File
-                           <i class="fa fa-fw  fa-upload"></i>
-                       </button>
-                       <input class="hide" type='file' id='uploadRulesFile' name='file'/>
-                       <button class="btn btn-primary download_rules mr-1 mt-1 d-inline-block">
-                           Download
-                           <i class="fa fa-fw  fa-download"></i>
-                       </button>
-	                   <button class="btn btn-primary download_rules_template mr-1 mt-1 d-inline-block">
-                           Get Template
-                           <i class="fa fa-fw  fa-download"></i>
-                       </button>
-                       <button class="btn btn-danger clear_all_rules mr-1 mt-1 d-inline-block" data-bind="click : $root.confirmClearAll.bind($data, 'rules')">
-                           Clear All
-                           <i class="fa fa-fw  fa-trash"></i>
-                       </button>
-	                </div>
-                </div>
-                <div class="col-12 pt-2">
-                    <h3>Rules</h3>
+<div class="container mw-100 pb-3" id="testRule">
```

Comment:
Yes I've known about this and it wasn't possible to keep making the window wider since you never know how deep an aggregation could become. Word wrapping can seem a bit weird though... I'm not sure how to solve this well.

Reply:
You can play with white-space property to see the results. I think most of them look either weird or could cause problems. pre-line keeps the formatting and tries to wrap lines that exceed the width. This is not always possible though as has been seen in the error message pop up. Some lines can't be naturally wrapped if they have no lines, spaces or commas etc. If the depth of the aggregation is too much there would also be many wrapped lines and could appear confusing.

Reply:
I'm not talking about the module showing the aggregated object, that one is fine. Maybe it's called something else, but I was referring to when you click the blu information marker at the bottom of the page a new window shows up with helpful tips about the page. This one could be expanded in width I assume, since the information wouldn't change in it.